### ISSUE TITLE
Can not detect kotlin-wrappers

### ISSUE THREAD

**Issue by mustafaozhan** [ben-manes/gradle-versions-plugin#442]
They are mostly used in Kotlin Multi Platform apps and the plugin can not detect them.
Maybe it is because they have weird version number like `17.0.0-pre.126-kotlin-1.4.10` please refere [here](https://github.com/JetBrains/kotlin-wrappers) for more detail.
```
Failed to determine the latest version for the following dependencies (use --info for details):
 - org.jetbrains:kotlin-react
 - org.jetbrains:kotlin-react-dom
 - org.jetbrains.kotlinx:kotlinx-html-js
```

**Comment by ben-manes** [comment]
I this with 0.34? If not, please try that version which contains a fix for multi-platform resolution

**Comment by mustafaozhan** [comment]
Yes exactly, I tried with latest (0.34.0)

**Comment by ben-manes** [comment]
hmm, we would need a sample project to debug with then.

**Comment by mustafaozhan** [comment]
You can clone this [repo](https://github.com/Book-Barter/BookBarter). Note that repo has version 0.33.0 currently please update it to 0.34.0. 
`buildSrc>src>main>kotlin>Configs.kt>Versions`
```
    const val versionChecker = "0.33.0" // this is the version for plugin
```

**Comment by ben-manes** [comment]
It does seem to be the same error that was fixed in 0.34, so not sure why it still occurs for you.

I'll try to take a deeper look later, but not sure when I'll find the time.

**Comment by mustafaozhan** [comment]
Yeap sure thank you :) just wanted to report it

**Comment by ben-manes** [comment]
In your case `commonMainMetadataElements` wasn't available to the JS project, so we couldn't capture the attributes. The `compile` configuration does exist and contain it, so used a restrictive fallback to copy those kotlin attributes if found. This seems to pass in your project, non-kotlin projects, and the projects from prior MPP issues.

**Comment by mustafaozhan** [comment]
Thanks a lot! I confirm that it is fixed with `0.35.0` 🎉
